radixdlt/babylon-wallet-android
# [ABW-4009] Filter out deleted entities in security problems

## Description
It does exactly what it says on the tin

## How to test

1. Bring your wallet in not recoverable state.
2. Delete one or more accounts
3. Verify the total amount of accounts that are not recoverable doesn't include deleted accounts


## PR submission checklist
- [X] I have tested security problems with deleted accounts.
